.github/workflows/ios.yml

```
@@ -26,14 +26,14 @@ concurrency:
 jobs:
   generate:
     name: e2e-${{ matrix.app_type }}-${{ matrix.jhipster_version }}
-    runs-on: macos-10.15
+    runs-on: macos-12
```

```suggestion
    runs-on: macos-11
```

mac os 11 has more available here https://github.com/actions/runner-images